packages/website-frontend/README.md

```diff
@@ -16,3 +16,80 @@ Run `npm run build` to build the project. The build artifacts will be stored in
 ## Running tests
 
 Run `npm run test` to execute the tests via [Vitest](https://vitest.dev/) and [Playwright](https://playwright.dev/).
+
+## Adding a report to the dashboard
+
+If you need to add a report quickly to the report, here is a simple CURL command that uploads one:
```

Comment:
You can refer to the dashboard docs: https://stryker-mutator.io/docs/General/dashboard/#send-a-report-via-curl